### PR TITLE
feat(cmx): add missing cluster create inputs to prepare command

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: prepare-cluster/action.yml [hunk 1]
```diff
@@ -21,25 +24,41 @@ inputs:
...
     description: 'Kubernetes version to provision (format is distribution dependent).'
     required: false
+  cluster-name:
+    description: 'Name of the cluster to provision'
+    required: false
+    default: ''
   ttl:
     description: 'Cluster TTL (duration, max 48h)'
     required: false
-  helm-values:
```
Comment (jdewinne) [inline prepare-cluster/action.yml:27]: Why remove this?

File: prepare-cluster/action.yml [hunk 2]
```diff
@@ -21,25 +24,41 @@ inputs:
...
-    description: 'The KOTS config values to use'
+  kubeconfig-path:
+    description: 'If specified, the kubeconfig will be written to this path'
     required: false
-  kots-wait-duration:
-    description: 'Timeout for KOTS to be used while waiting for individual components to be ready. must be in Go duration format (eg: 10s, 2m) (default "2m")'
+  export-kubeconfig:
+    description: 'Export the KUBECONFIG variable (true/false)'
     required: false
+    default: 'false'
```
Comment (jdewinne) [inline prepare-cluster/action.yml:61]: Shouldn't you also use these in the code of the action?

File: prepare-cluster/action.yml (hunk 1 above)
Comment (emosbaugh) [reply prepare-cluster/action.yml:27]: that was a mistake. fixed

File: prepare-cluster/action.yml (hunk 2 above)
Comment (emosbaugh) [reply prepare-cluster/action.yml]: there is no code is there?

**Review by jdewinne — APPROVED** [review]

**Pull request merged**